sky/server/constants.py

```diff
@@ -10,7 +10,7 @@
 # based on version info is needed.
 # For more details and code guidelines, refer to:
 # https://docs.skypilot.co/en/latest/developers/CONTRIBUTING.html#backward-compatibility-guidelines
-API_VERSION = 21
+API_VERSION = 22
```

Comment:
Looks like we still need to add logic to actually check if we are on the new server version?